src/network/networkd-bridge-mdb.c

```diff
@@ -355,10 +376,16 @@ int config_parse_mdb_group_address(
         if (r < 0)
                 return log_oom();
 
-        r = in_addr_from_string_auto(rvalue, &mdb->family, &mdb->group_addr);
-        if (r < 0) {
-                log_syntax(unit, LOG_WARNING, filename, line, r, "Cannot parse multicast group address: %m");
-                return 0;
+        r = parse_ether_addr(rvalue, &mdb->l2_addr);
+        if (r == 0)
+                mdb->type = BRIDGE_MDB_ENTRY_TYPE_L2;
+        else {
+                r = in_addr_from_string_auto(rvalue, &mdb->family, &mdb->group_addr);
+                if (r < 0) {
+                        log_syntax(unit, LOG_WARNING, filename, line, r, "Cannot parse multicast group address: %m");
```

Comment:
log message should be updated, to say that this could neither be parsed as mac address nor as ip address maybe?

Reply:
Agree, added further clues to error message